src/annotate/popv/config.vsh.yaml

```diff
@@ -0,0 +1,148 @@
+functionality:
+  name: popv
+  namespace: "annotate"
+  description: "Performs popular major vote cell typing on single cell sequence data https://github.com/czbiohub/PopV"
+  authors:
+    - __merge__: ../../authors/matthias_beyens.yaml
+      roles: [ author ]
+    - __merge__: ../../authors/robrecht_cannoodt.yaml
+      roles: [ author ]
+  argument_groups:
+    - name: Inputs
+      description: Arguments related to the input (aka query) dataset.
+      arguments:
+        - name: "--input"
+          alternatives: [-i]
+          type: file
+          description: Input h5mu file.
+          direction: input
+          required: true
+          example: input.h5mu
+        - name: "--modality"
+          description: List of modalities to process.
+          type: string
+          default: "rna"
+          required: false
+        - name: "--input_obs_batch"
+          type: string
+          description: Key in obs field of query adata for batch information.
+          required: false
+        - name: "--input_obs_layer"
+          type: string
+          description: If not None, will use data from query_adata.layers[query_layers_key].
+          required: false
+        - name: "--input_obs_labels"
+          type: string
+          description: Key in obs field of query adata for label information. This is only used for training scANVI. Make sure to set unknown_celltype_label to mark unlabelled cells.
+          required: false
+        - name: "--input_var_subset"
+          type: string
+          description: Subset the input object with this column.
+          required: false
+        - name: "--unknown_celltype_label"
+          type: string
+          description: If query_labels_key is not None, cells with label unknown_celltype_label will be treated as unknown and will be predicted by the model.
+          default: "unknown"
+          required: false
+    - name: Reference
+      description: Arguments related to the reference dataset.
+      arguments:
+        - name: "--reference"
+          type: file
+          description: "User-provided reference tissue. The data that will be used as reference to call cell types."
+          example: TS_Bladder_filtered.h5ad
+          direction: input
+          required: false
```

Comment:
Is the reference not required?

Reply:
Not if pretrained reference models are provided. However, I'm considering removing this feature for now.

Reply:
Not if user-provided models are passed. However, I disabled this option until it is needed in the future, so this parameter was now changed to `requred: true`.